lib/ttfunk/table/cff/charsets/expert.rb

```diff
@@ -0,0 +1,387 @@
+module TTFunk
+  class Table
+    class Cff < TTFunk::Table
+      module Charsets
+        EXPERT = OneBasedArray.new([
+          'space',
+          nil,
```

Comment:
You can save a couple of lines by using the following trick:
```
[
  * [nil] * 11
]
```

It's a combination of two separate things.
1. Array multiplication: `[nil] * 3` gives you `[nil, nil, nil]`
2. Splat operator. It's often used in method calls but also works in array literals. It basically unwrapps an array into a series of values: `[* [1, 2, 3]]` is the same as `[1, 2, 3]`.

I don't insist as these files are not going to be edited often. Mention because it's a neat trick and might be useful in cases like this.

Reply:
Very cool, thank you!